pkg/pillar/cmd/logmanager/logmanager.go

```diff
@@ -128,6 +129,13 @@ type zedcloudLogs struct {
 	LastSuccess  time.Time
 }
 
+type inputLogMetrics struct {
+	totalDeviceLogInput uint64
+	totalAppLogInput    uint64
+
+	sync.RWMutex
```

Comment:
Do you use this anywhere?

In general the CPU instructions to load and store 64 bit integers are atomic, so a mutex is only needed if people care about the consistency across a set of counters.

Reply:
Yeah, mutex looks unnecessary here. Will remove.

Reply:
done